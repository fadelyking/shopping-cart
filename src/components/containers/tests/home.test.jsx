import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import Homepage from "../home";

describe("Homepage", () => {
	it("Should render the homepage", () => {
		const { container } = render(
			<MemoryRouter>
				<Homepage />
			</MemoryRouter>
		);
		expect(container).toMatchSnapshot();
	});
});
