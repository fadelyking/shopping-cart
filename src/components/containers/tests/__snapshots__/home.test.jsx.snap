// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Homepage > Should render the homepage 1`] = `
<div>
  <div
    class="nav-bar"
  >
    <div
      class="nav-links"
    >
      <ul>
        <li>
          <a
            href="/"
          >
            <h1>
              Home
            </h1>
          </a>
        </li>
        <a
          href="/shop"
        >
          <li>
            Shop
          </li>
        </a>
      </ul>
    </div>
    <a
      href="/checkout"
    >
      <div
        class="cart"
      >
         Cart Icon
      </div>
    </a>
  </div>
  <div
    class="container"
  >
    <div
      class="home-container"
    >
      <div
        class="image-text"
      >
        <img
          src="forest.jpg"
        />
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ab non ipsam consequatur temporibus hic qui laborum impedit cumque! Placeat, vel? Assumenda, similique non. Rerum explicabo qui voluptate laboriosam tempore eveniet!
        </p>
      </div>
      <div
        class="btns"
      >
        <button
          class="primary"
        >
          Learn More !
        </button>
        <button
          class="primary"
        >
          Visit Shop
        </button>
      </div>
    </div>
  </div>
</div>
`;
